### DISCUSSION TITLE
Process for petitioning for or triggering or executing a backport/cherry-pick patch release?

### DISCUSSION THREAD

**Discussion by sthornington** [bazelbuild/bazel#24620] [Ideas]
Hi folks - I wrote https://github.com/bazelbuild/bazel/pull/24295 a few weeks ago, and I was somewhat dismayed to see that it was not included in the recent 8.0.0 release. This fix/feature is really important to us, and I’d like to get it into 8.0, if not also 7.4.  

I have read https://github.com/bazelbuild/continuous-integration/blob/master/docs/release-playbook.md but the link to the Release Managers Team at the top of the doc goes to a 404 for me.  I had kind-of assumed that actually executing a Bazel release was not something I myself could do, but I am absolutely willing to follow the checklist if it’s possible.  If it’s not possible, and official releases are only doable by affiliated team members, is there someone whom I could petition to do one for this one little fix?  It’s really very small, unlikely to need to change across all the branches (I checked when I wrote it, and actually developed it against 7.4 before submitting it on master as a PR), and it really means a lot to me to get this fixed once and for all.  

Please advise! 

Thank you,

Simon.

**Comment by meteorcloudy** [comment]
@sthornington The release playbook is more for Bazel release managers, for community member, please see https://bazel.build/release#release-procedure-policies.

Basically anyone can mark a certain PR or issue as a potential release blocker with `@bazel-io flag` so that it's on our radar. During preparing a release, you can also file a [cherry-pick](https://github.com/bazelbuild/bazel/issues/new?assignees=&labels=&projects=&template=cherry_pick_request.yml) request issue or send a PR to a release branch to backport a change that has already landed on the master branch.

We are not planning a Bazel 7.x release yet, but will start a 8.0.1 release soon, I think we could include https://github.com/bazelbuild/bazel/pull/24295.

**Reply by sthornington** [reply]
Thank you!  I will make the cherry-pick request now.

**Reply by sthornington** [reply]
Okay I just made https://github.com/bazelbuild/bazel/issues/24622 - I put @Wyverald as the reviewer since they reviewed the original PR.  Please let me know if there is anything else I should do.  I'm also happy to make cherry-pick PRs myself, but I don't see open branches for 7.4.2 nor 8.0.1 (yet).  Thanks.  EDIT: oh, hmm, I cannot set the Milestone thing.

**Reply by meteorcloudy** [reply]
Yeah, we haven't created the release branch for 8.0.1 yet, @iancha1992 will do that soon.

**Reply by sthornington** [reply]
Ok well I made the issue and flagged the pr with the bazel - io thing, I will keep my fingers crossed that it gets brought into 8.0.1.  Thanks again.

**Reply by iancha1992** [reply]
@sthornington The release branch (`release-8.0.1`) has been created now. Also the change is being cherry-picked to the branch in https://github.com/bazelbuild/bazel/pull/24630.
